test-it/ViaTests/InlineConstants.via

```diff
@@ -21,7 +21,7 @@ define(HelloWorld dv(.VirtualInstrument (
         it need to be escaped. How.
 */
 
-        StringToUpper(* str)
+        StringToUpper(str str)
```

Comment:
Are we modifying this because we are choosing to not update StringToUpper to handle null values at the moment?

Reply:
Sounds like this concern will be addressed as part of https://github.com/ni/VireoSDK/issues/358